sdk/deviceupdate/azure-deviceupdate/README.md

```diff
@@ -0,0 +1,47 @@
+# Device Update for IoT Hub client library for Python
+
+The library provides access to the Device Update for IoT Hub service that enables customers to publish updates for their IoT devices to the cloud, and then deploy these updates to their devices (approve updates to groups of devices managed and provisioned in IoT Hub). 
+
+  [Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/deviceupdate) | [Package (PyPI)](https://pypi.org/project/azure-deviceupdate/) | [Product documentation](https://github.com/Azure/adu-private-preview/blob/master/docs/adu-overview.md)
+
+## Getting started
+
+### Prerequisites
+
+- Microsoft Azure Subscription: To call Microsoft Azure services, you need to create an [Azure subscription](https://azure.microsoft.com/free/)
+- Device Update for IoT Hub instance
+- Azure IoT Hub instance
+- Python 2.7, or 3.5 or later is required to use this package.
+
+### Install the package
+
+Install the Device Update for IoT Hub client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-deviceupdate --pre
```

Comment:
based off talks with @lmazuel it seems you actually want your package to be azure iot deviceupdate. You should update the installation instructions in the readme here, but also change your [config file](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/deviceupdate/data-plane/readme.python.md) to have `package-name: azure-iot-deviceupdate` and `namespace: azure.iot.deviceupdate`